src/components/admin/AdminUserCard.vue

```diff
@@ -41,22 +41,18 @@ const checkRole = (item, role) => item.roles.find(i => i.role === role) ? true :
 
 <template>
     <div :class="['admin-user', props.className]">
+        <div class="admin-user__username">{{ user.username }}</div>
         <div class="admin-user__first-name">{{ user.firstName }}</div>
         <div class="admin-user__last-name">{{ user.lastName }}</div>
-        <div class="admin-user__username">{{ user.username }}</div>
         <div class="admin-user__email">{{ user.email }}</div>
         <div class="admin-user__created-at">
             {{ splitDate(user.createdAt)[1].slice(0, 5) }}
             <br>
             {{ splitDate(user.createdAt)[0] }}
         </div>
-        <div class="admin-user__updated-at">
-            {{ splitDate(user.updatedAt)[1].slice(0, 5) }}
-            <br>
-            {{ splitDate(user.updatedAt)[0] }}
-        </div>
         <div class="admin-user__role">
-            <ToggleButton :state="checkRole(user, 'ADMIN')" @click="updateRole(user)" />
+            ADMIN:
```

Comment:
I'm in doubts regarding placing the Admin in div, might want to wrap it in a label?